test/unit/configmap_spec.js

```diff
@@ -85,14 +90,14 @@ describe('Examine Config Maps',  function () {
             it('has mosquitto.conf', function () {
                 // need to check values
             })
-            it('has service', function () {
-                const service = services.filter( s => s.metadata.name === 'flowforge-broker')
-                service.should.have.length(1)
-            })
-            it('has ingress', function () {
-                const ingress = ingresses.filter( s => s.metadata.name === 'flowforge-broker')
-                ingress.should.have.length(1)
-            })
+            // it('has service', function () {
+            //     const service = services.filter( s => s.metadata.name === 'flowforge-broker')
+            //     service.should.have.length(1)
+            // })
+            // it('has ingress', function () {
+            //     const ingress = ingresses.filter( s => s.metadata.name === 'flowforge-broker')
+            //     ingress.should.have.length(1)
+            // })
```

Comment:
```suggestion
```

Looks like these have been moved to individual _spec files - remove commented code